lib/node_modules/@stdlib/stats/base/dists/pareto-type1/ctor/benchmark/benchmark.js

```diff
@@ -436,18 +518,23 @@ bench( pkg+':pdf', function benchmark( b ) {
 	var alpha;
 	var beta;
 	var dist;
+	var len;
 	var x;
 	var y;
 	var i;
 
 	alpha = 100.56789;
 	beta = 55.54321;
 	dist = new Pareto1( alpha, beta );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( EPS, 100.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( 0.0, 100.0 );
```
Same comment as above. If EPS is necessary here, it's fine to keep it. Otherwise, the range should remain the same as original.